superset-frontend/package-lock.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "superset",
   "version": "0.999.0dev",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
+<<<<<<< HEAD
```

Comment:
We don't want to update this file in a PR involving just the backend. You can rever this with:

```bash
$ git checkout master -- superset-frontend/package-lock.json
```